recipe/magento2.php

```diff
@@ -75,16 +75,18 @@
     'var/view_preprocessed/*'
 ]);
 
+set('bin/magento', '{{release_or_current_path}}/bin/magento');
+
 set('magento_version', function () {
     // detect version
-    $versionOutput = run('{{bin/php}} {{release_or_current_path}}/bin/magento --version');
+    $versionOutput = run('{{bin/php}} {{release_or_current_path}}/{{bin/magento}} --version');
```

Comment:
Isn't the `{{release_or_current_path}}` already added to `{{bin/magento}}` on line 78?

Reply:
Good catch, looks like I messed that up when resolving conflicts. I'll fix it